src/team.njk

```diff
@@ -0,0 +1,51 @@
+---
+title: Meet the Team
+layout: page
+---
+<div class="w-full bg-white">
+    {% if collections.posts.length > 0 %}
```

Comment:
We don't need the this `{% if collections... %}` bit - that's specific to the blog index page.